### PR TITLE
Fix split bugs

### PR DESCRIPTION
**Pull request by jacob314**
https://github.com/flutter/devtools/issues/914
https://github.com/flutter/devtools/issues/880

The core problem was we were attaching splitters
to elements not yet attached to the DOM resulting in
the splitter library suggesting negative values for
split percentages.

### CODE REVIEW

File: packages/devtools/pubspec.yaml [hunk 1]
```diff
@@ -35,9 +35,7 @@ dependencies:
     ^0.0.2
 #    path: ../../third_party/packages/primer_css
   split:
-    # TODO(devoncarew): We're pegged at 0.0.2 until we resolve layout issues with
-    # 0.0.4 (#880).
-    0.0.2
+    0.0.5
```
Comment (kenzieschmoll) [inline packages/devtools/pubspec.yaml]: has 0.0.5 been published yet?

File: packages/devtools/lib/src/timeline/timeline_screen.dart [hunk 2]
```diff
@@ -309,12 +308,15 @@ class TimelineScreen extends Screen {
     // Configure the flame chart / event details splitter if we haven't
     // already.
     if (!splitterConfigured) {
+      // TODO(jacobr): we need to tweak this layout so there is more room to
+      // display this UI. On typical devices, the space available is very
+      // limited making the UI harder to use than it would be otherwise./
```
Comment (kenzieschmoll) [inline packages/devtools/lib/src/timeline/timeline_screen.dart]: nit: remove slash at end of this line.

File: packages/devtools/lib/src/debugger/debugger.dart [hunk 3]
```diff
@@ -126,15 +126,16 @@ class DebuggerScreen extends Screen {
     return false;
   }
 
+  List<CoreElement> _navEditorPanels;
+  CoreElement _sourceArea;
+  CoreElement _consoleDiv;
+
```
Comment (kenzieschmoll) [inline packages/devtools/lib/src/debugger/debugger.dart]: nit: new lines between vars

File: packages/devtools/pubspec.yaml (hunk 1 above)
Comment (jacob314) [reply packages/devtools/pubspec.yaml]: just published it now :)

File: packages/devtools/lib/src/timeline/timeline_screen.dart (hunk 2 above)
Comment (jacob314) [reply packages/devtools/lib/src/timeline/timeline_screen.dart]: done

File: packages/devtools/lib/src/debugger/debugger.dart (hunk 3 above)
Comment (jacob314) [reply packages/devtools/lib/src/debugger/debugger.dart]: done. also moved to a better spot in the file.

**Pull request merged**